README.md

```diff
@@ -80,7 +80,7 @@ mvn clean package -Pspark-1.1 -Dhadoop.version=2.6.0 -Phadoop-2.6 -Pyarn -DskipT
 ```
 Yarn (Hadoop 2.7.x)
 ```
-mvn clean package -Pspark-1.1 -Dhadoop.version=2.7.0 -Phadoop-2.6 -Pyarn -DskipTests
```

Comment:
`-Phadoop-2.6` is enough to support hadoop 2.7.x for now like using `-Phadoop-2.4` for building it with hadoop 2.5